package.json

```diff
@@ -20,6 +20,7 @@
     "jest": "^29.3.1",
     "prettier": "^2.7.0",
     "pretty-quick": "^3.1.0",
-    "pro-commit": "^1.2.0"
+    "pro-commit": "^1.2.0",
+    "typescript": "^4.9.4"
```

Comment:
What is typescript used for here?

Reply:
Nevermind I see its required for the action

Reply:
It's apparently a peer dependency of `@typescript-eslint/parser` or something, as when it tried to run our tests previously it failed the it couldn't find the `typescript` module.